Detectors/TRD/workflow/src/trd-tracking-workflow.cxx

```diff
@@ -43,5 +46,11 @@ WorkflowSpec defineDataProcessing(ConfigContext const& configcontext)
   auto disableRootInp = configcontext.options().get<bool>("disable-root-input");
   auto disableRootOut = configcontext.options().get<bool>("disable-root-output");
   auto useTrackletTransformer = configcontext.options().get<bool>("use-tracklet-transformer");
-  return std::move(o2::trd::getTRDTrackingWorkflow(disableRootInp, disableRootOut, useTrackletTransformer));
+
+  auto wf = o2::trd::getTRDTrackingWorkflow(disableRootInp, disableRootOut, useTrackletTransformer);
+
+  // configure dpl timer to inject correct firstTFOrbit: start from the 1st orbit of TF containing 1st sampled orbit
+  o2::raw::HBFUtilsInitializer hbfIni(configcontext, wf);
```

Comment:
Just out of curiosity, why don't we write the ini for the TRD workflow?

Reply:
We have no particular policy to store the ini files, up to authors...